pkg/controllers/provisioning/v1alpha1/reallocation/controller.go

```diff
@@ -70,14 +70,14 @@ func (c *Controller) Reconcile(object controllers.Object) error {
 		return fmt.Errorf("listing underutilized nodes, %w", err)
 	}
 
-	// TODO: Further filter underutilized nodes that haven't been cordoned/TTLed to not spam logs
-	if len(underutilized) != 0 {
-		zap.S().Debugf("Found %d underutilized nodes", len(underutilized))
+	// Filter underutilized nodes to those that haven't had TTL set
+	nonTTLed := c.filter.GetTTLableNodes(underutilized)
```

Comment:
nit: try to match your variable names to the accessor functions unless you have a good reason not to (i.e. ttlableNodes)

Reply:
Good point. Thanks for the catch.